src/platforms/javascript/guides/angular/components/tracehelpers.mdx

```diff
@@ -4,7 +4,42 @@ title: Track Angular Components
 
 To track Angular components as part of your transactions, use any of these three options:
 
-1. `TraceClassDecorator` tracks a duration between `OnInit` and `AfterViewInit` lifecycle hooks in components:
+1. `TraceDirective` tracks a duration between `OnInit` and `AfterViewInit` lifecycle hooks in template:
+
+<Alert level="info" title="Compatibility">
```

Comment:
I wonder if this wouldn't fit better in the troubleshooting page. If it is really important that everybody sees this I would leave it here, if it's only relevant to a limited amount of people I'd move it to the troubleshooting page. I leave this up to you. :D

Reply:
That's a good point. I'll just move it there and in case people really complain (and don't find the info in Troubleshooting), we can put it back to this page again.

Reply:
Hmm it seems like I can't add anything to Troubleshooting without copying the entire document from `javascript/common` to `javscript/guides/angular`. Any ideas?
Tbh it doesn't seem worth the trouble to duplicate the entire document just for this paragraph. Debating whether to simply leave it in `tracehelpers.mdx`